src/djangosaml2_spid/views.py

```diff
@@ -94,62 +74,62 @@ def spid_sp_authn_request(conf, selected_idp, binding,
 
     # message id
     authn_req.id = saml2.s_utils.sid()
-    authn_req.version = saml2.VERSION # "2.0"
+    authn_req.version = saml2.VERSION  # "2.0"
     authn_req.issue_instant = saml2.time_util.instant()
 
     name_id_policy = saml2.samlp.NameIDPolicy()
-    # del(name_id_policy.allow_create)
-    name_id_policy.format = name_id_format # settings.SPID_NAMEID_FORMAT
-    authn_req.name_id_policy  = name_id_policy
+    name_id_policy.format = settings.SPID_NAMEID_FORMAT
+    authn_req.name_id_policy = name_id_policy
 
-    # settings.SPID_AUTH_CONTEXT
-    authn_context = requested_authn_context(class_ref=authn_context)
+    authn_context = requested_authn_context(class_ref=settings.SPID_AUTH_CONTEXT)
     authn_req.requested_authn_context = authn_context
 
-    # force_auth = true only if SpidL >= 2
```

Comment:
it should be dynamic, but in production forceAuthn = True resolved some issues ... please leave that comment as placeholder

Reply:
Sure! Is there some documentation about the issues, perhaps we can open one and make a better placeholder with a reference to it?

Reply:
if L > 1 then forceauthn must be True, you can comment out that behaviour, thank you

Reply:
@peppelinux Done! 👍🏻
Check it out and resolve the conversation.